tests/_support/index.js

```diff
@@ -108,11 +108,24 @@ Cypress.Commands.add("configureCluster", function(configuration) {
     }
   }
 
-  if (configuration.mesos === "1-pod") {
+  if (configuration.mesos === "1-pod" || "1-pod-delayed") {
```

Comment:
Here is the mistake that causes your PR to fail
```suggestion
  if (configuration.mesos === "1-pod" || configuration.mesos === "1-pod-delayed") {
```